src/igvfd/audit/measurement_set.py

```diff
@@ -182,3 +182,47 @@ def audit_preferred_assay_title(value, system):
             f'which is not an expected preferred assay title for this assay term.'
         )
         yield AuditFailure('inconsistent assay metadata', detail, level='WARNING')
+
+
+@audit_checker('MeasurementSet', frame='object')
+def audit_inconsistent_institutional_certification(value, system):
+    '''
+        audit_detail: Measurement sets for mapping assays are expected to link to samples covered by an institutional certificate issued to a matching lab and award.
+        audit_category: inconsistent institutional certification
+        audit_levels: ERROR
+    '''
+    # Characterization assays do not need to be audited.
+    characterization_assays = [
+        'OBI:0003133',  # cas mediated mutagenesis
+        'NTR:0000520',  # CRISPR screen
+        'OBI:0000916',  # flow cytometry assay
+        'OBI:0000185',  # imaging assay
+        'OBI:0002675',  # massively parallel reporter assay',
+        'OBI:0000288',  # protein-protein interaction detection assay',
+        'OBI:0002041'  # self-transcribing active regulatory region sequencing assay
+    ]
+    assay_term = value.get('assay_term')
+    assay_object = system.get('request').embed(assay_term, '@@object?skip_calculated=true')
+    assay_term_id = assay_object.get('term_id')
+    if assay_term_id in characterization_assays:
+        return
+
+    lab = value.get('lab')
+    award = value.get('award')
+    samples = value.get('samples')
```

Comment:
should not for each of these gets be a default value returned? 
samples = value.get('samples', [])
  lab = value.get('lab', None)
    award = value.get('award', None)